assets/scripts/palette/Palette.jsx

```diff
@@ -1,22 +1,17 @@
 import React, { useRef, useEffect } from 'react'
-import PropTypes from 'prop-types'
 import { IntlProvider } from 'react-intl'
 import { useSelector } from 'react-redux'
 import Scrollable from '../ui/Scrollable'
+import Tooltip, { useSingleton } from '../ui/Tooltip'
 import SegmentForPalette from '../segments/SegmentForPalette'
 import { getAllSegmentInfoArray } from '../segments/info'
 import { generateRandSeed } from '../util/random'
 import './Palette.scss'
 
-Palette.propTypes = {
-  handlePointerOver: PropTypes.func.isRequired,
-  handlePointerOut: PropTypes.func.isRequired,
-  handleScroll: PropTypes.func.isRequired
-}
-
 function Palette (props) {
   const flags = useSelector((state) => state.flags)
   const locale = useSelector((state) => state.locale)
+  const [source, target] = useSingleton()
```

Comment:
I'm going to add some more explanation (and link to docs) to the `<Tooltip />` component itself, because I'll assume that when you need the information it's going to be related to learning how `<Tooltip />` works. Adding a comment to `<Palette />` seems less efficient because, while it can be super convenient when reading through `<Palette />`, begs the question: do we add documentation about `useSingleton()` every time it is imported and used?